02_content/04_m-tag/01_inline-tag.md

```diff
@@ -4,12 +4,48 @@ title: Inline tag
 nav_order: 1
 grand_parent: Content
 parent: M-tag
-last_modified_date: 20-10-2021 13:00
+last_modified_date: 04-11-2021 14:00
 permalink: /content/m-tag/inline-tag/
 ---
 
 # Inline tag
 
+When creating content in Masa CMS you use the `[mura]` tag for rendering dynamic content.
+This is very powerfull way to acces CFML and Masa functionality.
+
 ## Examples
+You can use CFML functions inside your `[mura]` tag:
+
+    [mura]TimeFormat(Now(), 'hh:mm:ss tt')[/mura]
```

Comment:
use https://pmarsceill.github.io/just-the-docs/docs/ui-components/code/ for code examples

Reply:
Done